gradle/dependencies.gradle

```diff
@@ -34,13 +34,13 @@ versions += [
   junit: "4.12",
   lz4: "1.3.0",
   metrics: "2.2.0",
-  powermock: "1.6.3",
+  powermock: "1.6.4",
   reflections: "0.9.10",
   rocksDB: "3.10.1",
```

Comment:
This likely shouldn't be updated in this patch, but we are pretty far behind on RocksDB given how new the streams module is.